include/nbl/builtin/hlsl/spirv_intrinsics/glsl.std.450.hlsl

```diff
@@ -3,27 +3,25 @@
 
 #ifdef __HLSL_VERSION
 #include "spirv/unified1/GLSL.std.450.h"
-#endif
 
 namespace nbl 
 {
 namespace hlsl
 {
-#ifdef __HLSL_VERSION
 namespace spirv
 {
 
 //[[vk::ext_instruction(GLSLstd450FindILsb, "GLSL.std.450")]]
-//int FindILsb(int32_t val);
+//int findMSB(int32_t val);
 
 [[vk::ext_instruction(GLSLstd450FindUMsb, "GLSL.std.450")]]
-int FindSMsb(int32_t val);
+int findMSB(int32_t val);
 
 [[vk::ext_instruction(GLSLstd450FindUMsb, "GLSL.std.450")]]
-int FindUMsb(uint32_t val);
+int findMSB(uint32_t val);
```

Comment:
nooo, these will be machine generated soon, make sure you match the name of the enum minus the  `Op` or `GLSLstd450` prefix

Reply:
I was working on these in my branch, I can cherrypick and commit if you want to https://github.com/Devsh-Graphics-Programming/Nabla/blob/20b4e3af5c5d9dc09d4ed8dfecff4678c79b140e/include/nbl/builtin/hlsl/spirv_intrinsics/core.hlsl

Reply:
sure

Reply:
@Fletterio these need to be in the `GLSL.std.450.hlsl` header, i will copy your code and move them there so you can actually cherrypick from me

Reply:
https://github.com/Devsh-Graphics-Programming/Nabla/commit/b3fba1486f1ffcbc62b88ec71f3981b867bf040f

Reply:
actually don't cherry pick it, it will be on master soon